locales/hin.json

```diff
@@ -203,6 +203,13 @@
       "technicalDifficulty": {
         "title": "एक त्रुटि पाई गई!",
         "message": "आपके धैर्य के लिए धन्यवाद! हम अभी तकनीकी समस्याओं का सामना कर रहे हैं। कृपया बाद में पुनः प्रयास करें या आगे की सहायता के लिए व्यवस्थापक से संपर्क करें!"
+      },
+      "verificationFailed": {
+        "title": "एक त्रुटि पाई गई!",
+        "goBackButton": "वापस जाओ",
+        "ERR_GENERIC": "तकनीकी त्रुटि के कारण, हम कार्ड डाउनलोड करने में असमर्थ रहे।",
+        "NETWORK_ERROR": "अस्थिर इंटरनेट कनेक्शन के कारण, हम कार्ड डाउनलोड करने में असमर्थ थे। कृपया अपने इंटरनेट कनेक्शन की जाँच करें।",
```

Comment:
Why this error code `NETWORK_ERROR` is different than eng language. In eng it is `ERR_NETWORK`

Reply:
Updated